### DISCUSSION TITLE
Pressure drop across the micromodel

### DISCUSSION THREAD

**Discussion by foojan-Ka** [GeoChemFoam/GeoChemFoam#42] [Q&A] [answered]
Hello,

I hope you're doing well.

I have a question regarding the calculation of pressure drop across a micromodel. After running the 2-phase flow in Tutorial 02, I attempted to calculate the pressure drop at each timestep between the inlet and outlet. I extracted a selection from the inlet and outlet, and then tried to obtain the average pressure in each selection. However, since pressure can have positive or negative values in different parts, I'm unsure of the best way to calculate the pressure difference between the inlet and outlet.

I would greatly appreciate your assistance.

Looking forward to your response.

Best regards,
Foojan

**Comment by julienmaes** [comment, marked as answer]
Dear Foojan,

Calculating the pressure drop is an unanswered research question. This paper, from Raeini et al., gives some answer you might want to check: https://www.sciencedirect.com/science/article/pii/S0309170814001730?via%3Dihub

**Reply by foojan-Ka** [reply]
Dear Julien,

Thanks a lot for sending the paper.
